open-source.adoc

```diff
@@ -292,50 +290,50 @@ about to do better. A distilled version of that list is:
+https://github.com/bitcoin/bitcoin/blob/master/doc/dependencies.md[minimize its exposure to third party libraries]. Of course, you can't
+eradicate all external dependencies, otherwise you'd have to write everything by yourself,
+from font rendering to implementation of system calls.
 
 === Review
 
 This section is named "`Review`", rather than "`Code review`", because
-Bitcoin's security relies heavily on review on multiple levels, not
-just source code. Also, different ideas require review on different
-levels. For example a consensus rule change would require more review
-on more levels than a color scheme change, or a typo fix.
-
-An idea usually flows through several phases of discussion and review
-on its way to final adoption. Some of these phases are listed below:
-
-. An idea is posted on the bitcoin-dev mailing list
-. The idea is formalized in a Bitcoin Improvement Proposal (BIP)
-. The BIP is implemented in a pull request to Bitcoin Core
+Bitcoin's security relies heavily on review at multiple levels, not
+just source code. Moreover, different ideas require review at different
+levels: a consensus rule change would require a deeper review
+at more levels compared to a color scheme change or a typo fix.
+
+On its way to final adoption, an idea usually flows through 
+several phases of discussion and review. 
+Some of these phases are listed below:
+
+. An idea is posted on the Bitcoin-dev mailing list
+. The idea is formalized into a Bitcoin Improvement Proposal (BIP)
+. The BIP is implemented in a pull request (PR) to Bitcoin Core
 . Deployment mechanisms are discussed
 . Some competing deployment mechanisms are implemented in pull
 requests to Bitcoin Core
-. Pull requests are merged into master branch
+. Pull requests are merged to the master branch
 . Users choose whether to use the software or not
 
-In each of these phases people with different points of view and
+At each of these phases people with different points of view and
 backgrounds review the available information, be it the source code, a
-BIP, or a loosely described idea. The phases usually aren't performed
-in any strict top-down manner, multiple phases can happen
+BIP, or just a loosely described idea. The phases are usually not performed
+in any strict top-down manner, indeed multiple phases can happen
 simultaneously, and sometimes you go back and forth between
-them. Different people may also provide feedback in different phases.
+them. Different people may also provide feedback during different phases.
 
 One of the most prolific code reviewers on Bitcoin Core is Jon
 Atack. He wrote
 https://jonatack.github.io/articles/how-to-review-pull-requests-in-bitcoin-core[a
 blog post] about how to review pull requests in Bitcoin Core. He
 emphasizes that a good code reviewer focuses on how to best add value.
 
-[quote, Jon Atack on his blog, How to Review Pull Requests in Bitcoin Core]
+[quote, Jon Atack, How to Review Pull Requests in Bitcoin Core (2022)]
```

Comment:
I think it's mostly from 2020. Look at the page's git history: https://github.com/jonatack/jonatack.github.io/commits/master/articles/how-to-review-pull-requests-in-bitcoin-core.html